command/command_test.go

```diff
@@ -325,6 +325,75 @@ func Test_removeNewLineChanges(t *testing.T) {
 	}
 }
 
+func Test_FixIndentation(t *testing.T) {
```

Comment:
Can we add a few more test cases with different indentations?

Reply:
Added some